oriedita/src/main/java/oriedita/editor/drawing/Grid.java

```diff
@@ -340,6 +348,19 @@ public void draw(Graphics g, Camera camera, int p0x_max, int p0y_max, boolean co
             s_tv.set(camera.object2TV(s_ob));
             g.drawLine((int) s_tv.determineAX(), (int) s_tv.determineAY(), (int) s_tv.determineBX(), (int) s_tv.determineBY()); //直線
         }
+        
+        if(drawDiagonalGridlines && gridAngle != -90){
+            for (int i = grid_screen_a_min; i <= grid_screen_a_max; i++) {
+                s_ob.set(d_grid_ax * i + d_grid_cx * grid_screen_b_min + okx0,
+                        d_grid_ay * i + d_grid_cy * grid_screen_b_min + oky0,
+                        d_grid_ax * i + d_grid_cx * grid_screen_b_max + okx0,
+                        d_grid_ay * i + d_grid_cy * grid_screen_b_max + oky0);
+
+                s_tv.set(camera.object2TV(s_ob));
+                System.out.println("bmin,max: (" + grid_screen_b_min + ", " + grid_screen_b_max + ")");
```

Comment:
Printing too much here could slow down the drawing (same for the print statement lower down)

Reply:
aah, yes, this is left from debugging.